apps/judicial-system/backend/src/app/modules/case/interceptors/defendantIndictmentAccessed.interceptor.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  CallHandler,
+  ExecutionContext,
+  Injectable,
+  NestInterceptor,
+} from '@nestjs/common'
+
+import {
+  DefendantEventType,
+  isIndictmentCase,
+  isPrisonAdminUser,
+  User,
+} from '@island.is/judicial-system/types'
+
+import { DefendantEventLog, DefendantService } from '../../defendant'
+import { Case } from '../models/case.model'
+
+const hasValidOpenByPrisonAdminEvent = (
```

Comment:
Wanted to add a unit test only for this logic. Would it make sense to add it in the `apps/judicial-system/backend/src/app/modules/case/interceptors/test` and export this function directly from the interceptor (to reference in the target test) or extract it into a lib/utility file and have specific tests there for such small functions `apps/judicial-system/backend/src/app/modules/case/lib/test` 🤔

Reply:
The former would make perfect sense.